Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,158 @@
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_imeating",
+                    category=['you'],prompt="I'm Eating/Drinking",pool=True,unlocked=True))
+
+label monika_imeating:
+    if mas_getEV('monika_imeating').shown_count == 0:
+        m 1eua "You know, although I don't get hungry, I still miss enjoying good food."
+        m 4eub "Like Natsuki's cupcakes."
+        m 4eua "For a bunch of lines of code, they were pretty tasty."
+        m 3hua "Are you eating anything right now, [player]?"
+
+default persistent_.mas_food_database = {}
+
+default persistent_.mas_food_current = None
+
+init 1 python in mas_food:
+    food_db = dict()
+
+    TYPE_FOOD = 0
```

Comment:
I see you're using these to categorize the foods/drinks, though isn't it just more efficient to type in `category=['food']` or `category=['drink']` and then use that?